Modules/Scripted/RigidAlignmentModule/CMakeLists.txt

```diff
@@ -6,25 +6,21 @@ set(${MODULE_NAME}_TITLE "Rigid Alignment Module" CACHE STRING "Module title")
 set(${MODULE_NAME}_CATEGORY "Groups" CACHE STRING "Module category")
 
 set(MODULE_METADATA_SCRIPT
-  ${CMAKE_CURRENT_BINARY_DIR}/../${MODULE_NAME}Metadata/__init__.py
+  ${CMAKE_CURRENT_BINARY_DIR}/_${MODULE_NAME}Metadata/__init__.py
```

Comment:
Is the addition of the extra `_` intentional ?

Reply:
IF yes, consider providing some details in the commit message to justify the rename.

Reply:
Yes, it is playing on the `_` means private in Python from [PEP-8](https://peps.python.org/pep-0008/#descriptive-naming-styles). The python module is essentially private to the module.

I will say as such in the commit message.